src/web/core/src/main/java/org/geoserver/web/data/layer/CascadedWFSStoredQueryAbstractPage.html

```diff
@@ -18,7 +18,7 @@
 
 <wicket:fragment wicket:id="newPage">
   <div>
-    <wicket:message key="chooseNativeName">Layer name </wicket:message> <input type="text" wicket:id="nativeName"></input>
+    <wicket:message key="chooseNativeName">Layer name </wicket:message> <input type="text" wicket:id="nativeName">
```

Comment:
Training was a typo, updated my comment above. I am going through and removing whitespace locally.

Only one that has given me pause is:
```
<input type="radio">mode 1</input>
```

Being turned into:
```
<input type="radio">mode 1
```